gui-apps/hyprshot/hyprshot-9999.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 2024 Misha Zavertkin <misha.zavertkin@mail.ru>
+# Distributed under the terms of the ISC License
+
+EAPI=8
+
+DESCRIPTION="Hyprshot is an utility to easily take screenshot in Hyprland using your mouse"
+HOMEPAGE="https://github.com/Gustash/Hyprshot/"
+
+if [[ ${PV} = 9999* ]]; then
+	EGIT_REPO_URI="https://github.com/Gustash/Hyprshot.git/"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/Gustash/Hyprshot/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
```

Comment:
Remove the `src_unpack` function and instead set `S=${WORKDIR}/Hyprshot-${PV}` above the `KEYWORDS` line.

Reply:
Done as amendment

Reply:
Thanks a bunch!